lib/tasks/deploy.rake

```diff
@@ -73,6 +71,21 @@ namespace :deploy do
     puts "Build URL: #{response['build_url']}"
   end
 
+  def deploy_envs
+    return DEPLOY_ENVS unless ENV.key?('DEPLOY_ENVS')
+    # e.g., DEPLOY_ENVS=master:production,staging:staging,staging:preview
```

Comment:
I guess it's only open source because it's convenient.

Look at the third line of this file...
``` 
DEPLOY_USER = 'ad2gamesdeploy'
```

But it was just a suggestion, we don't have to change it.